harness/determined/common/experimental/user.py

```diff
@@ -5,6 +5,28 @@
 
 
 class User:
+    """
+    A User object represents an individual account on a Determined installation.
+    It can be obtained from ``client.list_users`` or ``client.get_user_by_name()``.
+
+    Attributes:
+        session: HTTP request session.
+        user_id: Unique ID for the user.
+        admin: (Mutable) Admin status of the user.
+        remote: (Mutable) Checkpoint associated with this model version.
+        agent_uid: (Mutable) Unix user ID on the agent the task runs on.
+        agent_gid: (Mutable) Unix group ID on the agent.
```

Comment:
I guess from this that users are associated with agents. Could the docstring describe that relationship?